compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -187,7 +190,12 @@ def __init__(
             warnings.warn(msg)
 
         # Execution backends
-        self.executors = executors  # List of executors
+        if executors is None:
+            global _DEFAULT_EXECUTORS
```

Comment:
Leaving a global var that is definitely not helping any user scenarios is a bad idea. Let's remove the global var entirely. It's not good for tests to rely on a hidden mechanism for better perf, when they could create an executor for reuse.

Reply:
Wasn't sure about the larger implications of that; have just removed it entirely.